test/verify/check-openshift

```diff
@@ -378,6 +378,34 @@ LABEL io.projectatomic.nulecule.atomicappversion="0.1.11" \
         b.click("#kubernetes-reconnect")
         b.wait_present("#service-list")
 
+    def bootstrapKubevirt(self):
+        o = self.openshift
+
+        o.execute("/root/nested-kvm") # not enabled by default
+        o.execute("(cat /sys/module/kvm_intel/parameters/nested || cat /sys/module/kvm_amd/parameters/nested) | grep -q Y")
```

Comment:
This is a nice validation. But this could actually go into test/verify/nested-kvm itself? (Fine for a follow-up PR, though).